File: coding_ninjas/Array/Methods/total_prime.java
/*
 Write a program to find the total number of a number of a prime number in given interval. 
 Given two integers S and E, count all primes between S and E.
*/

import java.util.* ;
import java.io.*; 
import java.util.Scanner;
class CountPrime 
{   
    int count=0;
    int totalPrime(int start, int end)
    {
    for(int i=start;i<=end;i++)
        {
            int count1=0;
            for(int j=1;j<=i;j++)
            {
                if(i%j==0)
                count1++;
            }
            if(count1==2)
            count++;
        }
        return count;
    }   
}

class total_prime {
	public static void main(String args[]) {
		
		CountPrime obj = new CountPrime();
		Scanner sc = new Scanner(System.in);
		int s = sc.nextInt();
		int e = sc.nextInt();
		System.out.println(obj.totalPrime(s, e));
	}
}

File: coding_ninjas/Python/totalPrime.py
'''
Write a program to find the total number of a primes number in a given interval. Given two integers S and E, count all primes between S and E.
'''

from os import *
from sys import *
from collections import *
from math import *




#Write your totalPrime function here.
def totalPrime(S, E):
    count = 0;
    for n in range(S, E+1):
        if n > 1:
            for i in range(2, n):
                if(n % i) == 0:
                    break
            else:
                    count += 1
    return count



#Taking S and E space seperated input.
S,E = map(int,input().split(' '))
print(totalPrime(S,E))